File: CPP/grassseed.cpp
// Kattis - grassseed
// Cliff Sparks

#include <iostream>
#include <cmath>
#include <iomanip>

using namespace std;

int main()
{
    cout << fixed << setprecision(7);
    float cost, l, w; 
    int lawns;
    float total, temp;
    int i;
    cin >> cost >> lawns;

    for (i = 0; i < lawns; i++)
    {
        cin >> l >> w;
        temp = l * w;
        total += temp; 
    }
    total *= cost;
    cout << total;

    return 0;
}

File: PY/grassseed.py
# grassseed
# Cliff Sparks

# RUNTIME Error

cost = float(input())
lawns = int(input())
temp = int(0)
total = int(0)
grand = int(0)
for i in range(lawns): 
    length, width = input().split()
    total += length * width
grand = total * cost
print("{:.7f}".format(grand))
